docs/src/components/HeadingCopyButton.astro

```diff
@@ -22,29 +22,38 @@
       }
 
       h.appendChild(copyButton)
+      h.classList.add('with-copy')
     })
-    window.ADD_COPY_BUTTONS()
+  window.ADD_COPY_BUTTONS()
 </script>
 <style is:global>
   button.copy:before {
     content: unset;
   }
   button.copy {
-    display: inline;
+    display: none;
```

Comment:
Make it transparent by default to made it focusable from a keyboard ("tab") and add `:focus` styles like `:hover`.

Reply:
Nice suggestion. Done!